### DISCUSSION TITLE
Allow for multiple selections

### DISCUSSION THREAD

**Discussion by haberdashPI** [community/community#6528] [Projects and Issues] [answered]
Being able to select multiple rows and modify their rows (e.g. set the status for all of them) would be great.

**Comment by ShrutiC-git** [comment]
I too think that having multi-select option within both the table view and board view would be very useful for assigning roles/changing status OR for deleting items.

**Comment by ams-ryanolson** [comment]
One of the things I find hard is when you are setting up a project and you cannot bulk assign tasks to a user or repo.

**Reply by GRCAINC** [reply]
As we tried to find solutions to multiple projects economic development, incubator, entrepreneurs, historical preservation, (history culture of the African-American)hydroponic garden conservation, social justice our community have been been left behind forgotten

**Comment by timoyates** [comment, marked as answer]
Hi folks, bulk actions are somewhat limited at this very moment (it's on our list!) but you can indeed bulk-archive and bulk-delete already. In order to multi-select items, be sure to shift/cmd/cntrl click on the rows via the row numbers on the far left side - that's the click target for multi-select. Cheers!

**Reply by kiview** [reply]
Are there plans to allow multi-move or e.g. bulk-edit for the Iteration field as well?

**Reply by angalic** [reply]
This is by far the biggest issue for me with github projects. I must be able to multiselect and reorder aswell ass multiselect and edit fields. The other functionality in projects is on point and sufficient for a small to medium sized project.

**Reply by yuseferi** [reply]
any update after a year?

**Reply by ADTC** [reply]
Is there any plan to allow "Convert to issue" across a selection of multiple issues? (obviously to the same repository) @timoyates 

I had to click "Convert to issue" one by one on 20 draft issues because neither the board nor the table allowed me to do it for all of them in one click.

**Comment by peterpolman** [comment]
Good stuff team!👏  But for me +1 for moving multiple selected items up or down in table view 😅

**Reply by pmccomb** [reply]
Yes!

**Comment by dralley** [comment]
This is something we really miss from the tracker we used until recently (Redmine).  It's a big timesaver to select multiple issues and tag them / put them on the sprint / close them in bulk.

**Comment by aogz** [comment]
+1 here. Would be really helpful to select multiple items and move them to the next iteration or tag them

**Reply by zhx828** [reply]
Second this. We recently moved from ZenHub, without this feature is driving the project manager(me) crazy. 😞

**Comment by lukeramsden** [comment]
No update on this? Very disappointing.

**Comment by angalic** [comment]
+1

**Comment by cjedlio** [comment]
+1

**Comment by fabien-roy** [comment]
Would seriously help managing projects. GH Projects are great, this would be a very helpful feature.

**Comment by yuseferi** [comment]
+1

**Comment by equezada-jej** [comment]
+1

**Comment by JVimes** [comment]
I still can't drag multi-selected cards around on Boards. I'm not sure Boards view is well supported. I hope it becomes more viable soon. Thank you.

**Comment by petrklapka** [comment]
Just adding another voice to the choir.  Using projects as a team dashboard and really need to "bulk change" custom and built in fields.

**Comment by bryanwhiting** [comment]
+1 to this →  First thing I googled when starting with github projects. Had to manually assign each one to the same person.

**Comment by bryanwhiting** [comment]
At least you can multi-assign in the github issues.

**Comment by sweepy84** [comment]
+1 Need to be able to drag & drop issues across board. Wasting a lot of time moving things around one by one.

**Comment by fdjkgh580** [comment]
Are you pressing shift/cmd/cntrl at the same time? I tried many times, not every time I can trigger successfully, it is not very good to use this operation.

**Comment by henrikvilhelmberglund** [comment]
It's possible to select them by shift clicking the numbers at the left but a bit pointless when you can't do much with the selection anyway (like converting to issues/setting values).

**Comment by iamWing** [comment]
Need drag and drop on multiple items. Missing this particularly when moving items to the next iteration.

**Comment by adigau** [comment]
Being able to apply a change to a group of issues in a project, based on a filter, would be very important 🙏

**Comment by petrklapka** [comment]
Just bumping this as it's been a while.  Several teams at Elastic are now using GH Projects such that 1 project = 1 team.  The Project has become our (team leads) home base for our team.  We're using custom fields of "Projects" and "Epics" to organize our teams work into project tabs, and epic groups.  This works great, but not being able to bulk change custom field values across multiple issues makes it a real chore.  Better UX for more view tabs/tabs with long names would also help this use case.

**Comment by brayhite** [comment]
+1 and seconding @petrklapka's use case. We have started to follow a similar 1 project = 1 team where each issue represents a chunk of work, and custom fields help organize the chunks into larger efforts/epics. Bulk editing to set an epic, update statuses, set assignees, etc. would be very welcome to help speed up the administrative bits of this.
